lib/HTTP/Status.pm

```diff
@@ -260,6 +272,15 @@ that it has erred or is incapable of performing the request.
 
 This function is B<not> exported by default.
 
+=item is_cacheable_by_default( $code )
+
+Return TRUE if C<$code> indicates that a response is cacheable by default, and
+it can be reused by a cache with heuristic expiration. All other status codes
+are not cacheable by default. See L<RFC 7231 - HTTP/1.1 Semantics and Content, 
+Section 6.1. Overview of Status Codes>
```

Comment:
It's nice to add link to https://tools.ietf.org/html/rfc7231#section-6.1.
```diff
diff --git lib/HTTP/Status.pm lib/HTTP/Status.pm
index 133372c..8677600 100644
--- lib/HTTP/Status.pm
+++ lib/HTTP/Status.pm
@@ -277,7 +277,7 @@ This function is B<not> exported by default.
 Return TRUE if C<$code> indicates that a response is cacheable by default, and
 it can be reused by a cache with heuristic expiration. All other status codes
 are not cacheable by default. See L<RFC 7231 - HTTP/1.1 Semantics and Content, 
-Section 6.1. Overview of Status Codes>
+Section 6.1. Overview of Status Codes|https://tools.ietf.org/html/rfc7231#section-6.1>.
 
 This function is B<not> exported by default.
 ```